pkg/nfd-client/worker/nfd-worker.go

```diff
@@ -54,7 +54,7 @@ import (
 	_ "sigs.k8s.io/node-feature-discovery/source/usb"
 )
 
-// Global config
+// NFDConfig is a Global config
```

Comment:
Hmm, the word *global* doesn't make much sense anymore, I think

```suggestion
// NFDConfig contains the configuration settings of NfdWorker.
```

Reply:
changed